pkg/datasource/csvsource/read_klines_test.go

```diff
@@ -0,0 +1,14 @@
+package csvsource
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReadKlinesFromCSV(t *testing.T) {
+	prices, err := ReadKlinesFromCSV("./testdata/BTCUSDT-1h-2021-Q1.csv", time.Hour)
+	assert.NoError(t, err)
+	assert.Len(t, prices, 2158)
```

Comment:
can you verify the prices, like check the prices of prices[0]

Reply:
definitely